docs/porting/libraries.md

```diff
@@ -1,5 +1,304 @@
+
+The first thing to understand about porting a library to .NET Core is that you must target a version of the [.NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md).  In fact, it's accurate to say that targeting .NET Core means targeting a version of the .NET Platform Standard.
+
+This means that you'll have to make a tradeoff between APIs you can use and platforms you can support, and pick the version of the .NET Platform standard that best suits the tradeoff you wish to make.
+
+As of right now, there are 6 different versions to consider: .NET Platform Standard 1.0 through 1.5.  If you pick a higher version, you get access to more APIs at the cost of running on less targets.  If you pick a lower version, your code can run on more targets but at the cost of less APIs available to you.
+
+A key thing to understand is that **a project targeting a lower version cannot reference a project targeting a higher version**.  For example, a project targeting the .NET Platform Standard version 1.2 cannot reference projects that target .NET Platform Standard version 1.3 or higher.  Projects **can** reference lower versions, though, so a project targeting .NET Platform Standard 1.3 can reference a project targeting .NET Platform Standard 1.2 or lower.
+
+It's recommended that you pick a single .NET Platform Standard version and use that throughout your library.
+
+Read more in [The .NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md).
+
+## .NET Core project model overview
+
+.NET Core introduces a new project model that you may wish to spend some time understanding before you begin porting your code.  Rather than provide a comparison with the .NET Framework project model, this section covers the essential pieces.
+
+### The project file: `project.json`
+
+.NET Core projects are defined by a directory containing a `project.json` file.  This file is where aspects of the project are declared, such as package dependencies, compiler configuration, runtime configuration, and more.
+
+The `dotnet restore` command reads this project file, restores all dependencies of the project, and generates a `project.lock.json` file.  This file contains all the necessary information the build system needs to build the project.
+
+To learn more about the `project.json` file, read the [.NET Core project system docs]((../project-model/overview.md).
+
+### The solution file: `global.json`
+
+The `global.json` file is an optional file to include in a solution which contains multiple projects.  It typically resides in the root directory of a set of projects.  It can be used to inform the build system of different subdirectories which can contain projects.  This is for larger systems composed of several projects.
+
+To learn more about the `global.json` file, check out the [.NET Core project system docs](../project-model/global-json-reference.md).
```

Comment:
for MSDN, we usually follow the style:
"For more information on/about something, see X". Not sure if we want to adopt this here for consistency across docs.